src/util.js

```diff
@@ -74,18 +74,25 @@ util.getAttrs = function( tagMarkup, settings )
     }
 };
 
-util.getRemote = function( uri, callback, toDataUri )
+function getRemote( uri, settings, callback, toDataUri )
 {
     if( /^\/\//.test( uri ) )
     {
         uri = "https:" + uri;
     }
 
+    var requestOptions = {
+        uri: uri,
+        encoding: toDataUri ? "binary" : ""
+    };
+
+    if( typeof settings.requestTransform === "function" )
+    {
+        requestOptions = settings.requestTransform( requestOptions ) || requestOptions;
```

Comment:
I don't favor functions where the return value is only sometimes meaningful, that tends to lead to confusion in my experience. How about just requiring the transformed object to be returned, so the above would be:

```js
requestTransform: function(options)
{
    options.qs = {
        foo: "bar"
    };
    return options;
}
```

And if `undefined` or `false` gets returned (or anything falsey would be ok) then that cancels the request? That way the interface is such that the returned value is always meaningful, instead of just sometimes. In the simplest case, where there is no transform, that just requires

```js
requestTransform: function(options) { return options; }
```

not that there would be a reason to do that, but it isn't particularly more complicated than the same function without any return statement and keeps the interface more consistent.